data/mods/stats_through_kills/modinfo.json

```
@@ -0,0 +1,19 @@
+[
+  {
+    "type": "MOD_INFO",
+    "ident": "stats_through_kills",
+    "name": "Stats Through Kills",
+    "authors": [ "KorGgenT" ],
+    "maintainer": [ "KorGgenT" ],
+    "description": "You gain XP from kills that you can spend on increasing your stats.",
+    "category": "rebalance",
+    "dependencies": [ "dda" ]
+  },
+  {
+    "type": "EXTERNAL_OPTION",
+    "name": "STK",
```

Change option name to reflect name change in `data/json/game_balance.json`
```suggestion
    "name": "STATS_THROUGH_KILLS",
```